src/background/redirects.js

```diff
@@ -25,10 +24,26 @@ function redirectFromGoogle(requestDetails) {
         .filter(qParam => qParam !== "" && !["poe", "wiki", "poewiki"].includes(qParam))
         .join("+")
 
+    let searchEngine = null;
+
+    switch (url.host) {
+        case "google.com":
+        case "www.google.com":
+            searchEngine = 'https://www.google.com/search?q=site:poewiki.net+'
+            break;
+        case "duckduckgo.com":
+        case "www.duckduckgo.com":
+            searchEngine = 'https://www.duckduckgo.com/?q=site:poewiki.net+'
+            break;
+        default: searchEngine = 'https://www.duckduckgo.com/?q=site:poewiki.net+'
```

Comment:
nit: you could just initialise `searchEngine` with the default rather than setting it here, you could even rearrange the fall-throughs like this:

```js
        default:
        case "duckduckgo.com":
        case "www.duckduckgo.com":
            searchEngine = 'https://www.duckduckgo.com/?q=site:poewiki.net+'
 ```

Reply:
Fixed in d512a69 , our fallback is google now - I've removed the default all-together as it serves no purpose anymore.